src/main/scala/nl/biopet/bioconda/BiocondaRecipe.scala

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2018 Sequencing Analysis Support Core - Leiden University Medical Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package nl.biopet.bioconda
+
+import org.yaml.snakeyaml.Yaml
+import java.io.{File, PrintWriter}
+
+import scala.collection.JavaConverters._
+import scala.io.Source
+
+class BiocondaRecipe(name: String,
+                     version: String,
+                     command: String,
+                     sourceUrl: String,
+                     sourceSha256: String,
+                     runRequirements: Seq[String],
+                     buildRequirements: Seq[String],
+                     testCommands: Seq[String],
+                     homeUrl: String,
+                     license: String,
+                     summary: String,
+                     defaultJavaOptions: Seq[String],
+                     buildNumber: Int = 0,
+                     notes: Option[String] = None) {
+
+  def fileName: String = sourceUrl.split("/").last
+  def wrapperFilename: String = s"$name.py"
+
+  def stringToFile(string: String, file: File): Unit = {
```

Comment:
biopet CommenUtils already have a method like this

Reply:
I am not going to import common utils as a dependency for 5 lines of code.